packages/cardano-services-client/scripts/createVersionSource.js

```diff
@@ -22,21 +22,39 @@ const apiVersion = Object.fromEntries(
   ])
 );
 
-const versionFile = path.join(__dirname, '../version.json');
+const supportedVersionsFile = path.join(__dirname, '../supportedVersions.json');
+const sameMajorVersion = (a: string, b: string): boolean => {
```

Comment:
Doesn't look like this would work when used like `previouslySupported.filter(sameMajorVersion(version))`
```suggestion
const sameMajorVersion = (a: string) => (b: string) => {
```

Reply:
@mkazlauskas fixed in 29f24e50a834dfcfbbb8b632ee9957fcb3700833